### ISSUE TITLE
output new dimension isn't readable

### ISSUE THREAD

**Issue by na-bot-o** [llooker/pylookml#35]
Output Format of new dimension made by addDimension method is incorrect indent

・Code
 ```
 view = lookml.VIew('pylookml')
 view.addDimension('id')
 view.id.sql = '${TABLE}.id'
 file = lookml.File('lookml_file')
 file.put(view)
```

・Expected Output
```
  dimension: id {
     type: string
     sql: ${TABLE}.id ;;
  }
```

・Result Output
```
    dimension: id {    type: string
    sql: ${TABLE}.id ;; }
```

**Comment by russlooker** [comment]
Thank you for the feedback! I implemented the fix and will be pushing it with the 2.0 release coming very soon

**Comment by russlooker** [comment]
The whitespace has been completely fixed in version 3.0.0, now available in pip
try out the new version and let me know any thoughts
